modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Rod (desynth)' AND ID = 19906;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Rod' AND ID = 11505;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Knight\'s Belt +1' AND ID = 42015;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Knight\'s Cloak +1' AND ID = 33039;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Knight\'s Mufflers +1' AND ID = 13049;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Knight\'s Sollerets +1' AND ID = 13040;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Squire\'s Robe +1' AND ID = 32031;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Squire\'s Shield +1' AND ID = 42517;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Royal Swordsman\'s Blade +1' AND ID = 12537;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Ruby Earring' AND ID = 24024;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Ruby Earring' AND ID = 24071;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Ruby Ring' AND ID = 24532;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Ruby Ring' AND ID = 24573;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Rugged Bugard Leather' AND ID = 41014;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Rugged Bugard Leather' AND ID = 41015;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Runner\'s Belt' AND ID = 42006;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sagacious Brocade Obi' AND ID = 34024;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sagacious Gold Obi' AND ID = 33523;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sai' AND ID = 13004;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sainti' AND ID = 23034;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Samsonian Belt' AND ID = 42007;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='San d\'Orian Helm' AND ID = 22529;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='San d\'Orian Mufflers' AND ID = 22520;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='San d\'Orian Sollerets' AND ID = 22501;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sapphire Earring (desynth)' AND ID = 24029;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sapphire Earring' AND ID = 24032;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sapphire Earring' AND ID = 24066;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sapphire Ring' AND ID = 24530;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sapphire Ring' AND ID = 24572;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sarcenet Cape' AND ID = 33028;
```

Comment:
Archive Capture fails to load, but comments suggest Zilart https://ffxi.allakhazam.com/db/item.html?fitem=4294